uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/zones/OrchestratorZoneControllerMockMvcTests.java

```diff
@@ -166,10 +166,12 @@ void testGetZone_Unauthorized(String url) throws Exception {
     void testGetZone_nameRequiredError(String url) throws Exception {
         performMockMvcCallAndAssertError(get(url), status().isBadRequest(),
                                          JsonUtils.writeValueAsString(
-                                             new OrchestratorErrorResponse("Required request parameter 'name' for " +
-                                                                           "method parameter type String is not " +
-                                                                           "present")),
-                orchestratorZonesReadToken);
+                                             new OrchestratorZoneResponse(null, null,
```

Comment:
We could work toward gradually replacing performMockMvcCallAndAssertError with performMockMvcCallAndAssertResponse. There is no need to have both.

I changed it in the next commit.